ci/do_ci.sh

```diff
@@ -449,16 +449,21 @@ elif [[ "$CI_TARGET" == "deps" ]]; then
   # Validate dependency relationships between core/extensions and external deps.
   ./tools/dependency/validate_test.py
   ./tools/dependency/validate.py
+
+  ls /usr/bin/python*
+  ls /usr/local/bin/python* || :
+
   # Validate the CVE scanner works. We do it here as well as in cve_scan, since this blocks
   # presubmits, but cve_scan only runs async.
-  python3.8 tools/dependency/cve_scan_test.py
+  bazel run --incompatible_use_python_toolchains=false --python_top=//tools:python3.8 "${BAZEL_BUILD_OPTIONS[@]}" //tools/dependency:cve_scan_test
```

Comment:
this is not ideal. it should either specify the toolchain properly (https://github.com/bazelbuild/bazel/issues/7899) or perhaps address why there is multiple python versions - ideally everything should use the same (newer) interpreter